types/underscore/underscore-tests.ts

```diff
@@ -511,6 +511,27 @@ _.chain([
     .sample()
     .value();
 
+// $ExpectType number
+_.chain([1, 2, 3, 4, 5, 6])
+    .chunk(3)
+    .first()
+    .compact()
```

Comment:
Oops, this `compact` call isn't actually necessary, I could have added this with test in the `first` PR.

Reply:
Oh, also I already did this test apparently with `compact` removed, so never mind, I'll just remove it.